### PR TITLE
chore: upgrade forc 0.60.0

### PR DESCRIPTION
**Pull request by Torres-ssf**
- Close: https://github.com/FuelLabs/fuels-ts/issues/2400

### CODE REVIEW

File: packages/abi-typegen/test/fixtures/templates/predicate-with-configurable/factory.hbs [hunk 1]
```diff
@@ -78,7 +78,7 @@ const _abi = {
         "type": 2,
         "typeArguments": null
       },
-      "offset": 560
+      "offset": 1152
```
Comment (danielbate) [inline packages/abi-typegen/test/fixtures/templates/predicate-with-configurable/factory.hbs:81]: How much bigger are the contract sizes? jw as the offset has doubled 😬

File: .changeset/six-bobcats-impress.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/versions": patch
+"@internal/forc": patch
```
Comment (nedsalk) [inline .changeset/six-bobcats-impress.md:3]: There shouldn't be a need for this package in the changesets anymore. We should ignore it.

**Review by nedsalk — APPROVED** [review]

File: packages/abi-typegen/test/fixtures/templates/predicate-with-configurable/factory.hbs (hunk 1 above)
Comment (Torres-ssf) [reply packages/abi-typegen/test/fixtures/templates/predicate-with-configurable/factory.hbs:81]: @danielbate How can I verify that?

Comment (danielbate) [reply packages/abi-typegen/test/fixtures/templates/predicate-with-configurable/factory.hbs:81]: Before/after the file size of the bin file. Tbh we can't do anything about this so this won't be a blocker.

Comment (Torres-ssf) [reply packages/abi-typegen/test/fixtures/templates/predicate-with-configurable/factory.hbs:81]: @danielbate Based on the example that I picked, it appears they've increased ~ 18%

**Pull request merged**